lib/apps/client.rb

```diff
@@ -0,0 +1,83 @@
+# frozen_string_literal: true
+
+require 'common/client/base'
+require 'common/client/concerns/monitoring'
+require_relative 'configuration'
+require_relative 'responses/response'
+require 'cgi'
+
+module Apps
+  # Proxy Service for Apps API.
+  class Client < Common::Client::Base
+    include SentryLogging
+    include Common::Client::Concerns::Monitoring
+
+    configuration Apps::Configuration
+
+    STATSD_KEY_PREFIX = 'api.apps'
+
+    attr_reader :search_term
+
+    def initialize(search_term = nil)
+      @search_term = search_term
+    end
+
+    # Get all apps
+    #
+    def get_all
+      with_monitoring do
+        raw_response = perform(:get, 'directory', nil)
+        Apps::Responses::Response.new(raw_response.status, raw_response.body, 'apps')
+      end
+    rescue => e
+      handle_error(e)
+    end
+
+    # Get an individual app
+    #
+    def get_app
+      with_monitoring do
```

Comment:
this is put here for `app_name`s that have whitespace in them, like `Apple Health`.